[Bug]: @tiptap/extension-text-style is suddenly required in 2.8.0

### Affected Packages

extension-bullet-list

### Version(s)

2.8.0

### Bug Description

Uncaught Error: Cannot find module '@tiptap/extension-text-style'

### Browser Used

Chrome

### Code Example URL

_No response_

### Expected Behavior

No new dependencies are needed, or if they are they are automatically managed.

### Additional Context (Optional)

_No response_

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

Also seeing this. Here's our error (nextjs):

```
Module not found: Can't resolve '@tiptap/extension-text-style'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../node_modules/@tiptap/starter-kit/dist/index.js
```

--- a/.changeset/light-pens-end.md
+++ b/.changeset/light-pens-end.md
@@ -1,0 +1,6 @@
+---
+"@tiptap/extension-ordered-list": patch
+"@tiptap/extension-bullet-list": patch
+---
+
+This resolves an issue where the bullet-list and ordered-list extensions were depending on the list-item and text-style extensions unneccesarily. They are no longer imported and constants are used instead.

--- a/packages/extension-bullet-list/package.json
+++ b/packages/extension-bullet-list/package.json
@@ -29,14 +29,10 @@
     "dist"
   ],
   "devDependencies": {
-    "@tiptap/core": "^2.9.0",
-    "@tiptap/extension-list-item": "^2.9.0",
-    "@tiptap/extension-text-style": "^2.9.0"
+    "@tiptap/core": "^2.9.0"
   },
   "peerDependencies": {
-    "@tiptap/core": "^2.7.0",
-    "@tiptap/extension-list-item": "^2.7.0",
-    "@tiptap/extension-text-style": "^2.7.0"
+    "@tiptap/core": "^2.7.0"
   },
   "repository": {
     "type": "git",

--- a/packages/extension-bullet-list/src/bullet-list.ts
+++ b/packages/extension-bullet-list/src/bullet-list.ts
@@ -1,6 +1,7 @@
 import { mergeAttributes, Node, wrappingInputRule } from '@tiptap/core'
-import { ListItem } from '@tiptap/extension-list-item'
-import { TextStyle } from '@tiptap/extension-text-style'
+
+const ListItemName = 'listItem'
+const TextStyleName = 'textStyle'
 
 export interface BulletListOptions {
   /**
@@ -86,7 +87,7 @@ export const BulletList = Node.create<BulletListOptions>({
     return {
       toggleBulletList: () => ({ commands, chain }) => {
         if (this.options.keepAttributes) {
-          return chain().toggleList(this.name, this.options.itemTypeName, this.options.keepMarks).updateAttributes(ListItem.name, this.editor.getAttributes(TextStyle.name)).run()
+          return chain().toggleList(this.name, this.options.itemTypeName, this.options.keepMarks).updateAttributes(ListItemName, this.editor.getAttributes(TextStyleName)).run()
         }
         return commands.toggleList(this.name, this.options.itemTypeName, this.options.keepMarks)
       },
@@ -111,7 +112,7 @@ export const BulletList = Node.create<BulletListOptions>({
         type: this.type,
         keepMarks: this.options.keepMarks,
         keepAttributes: this.options.keepAttributes,
-        getAttributes: () => { return this.editor.getAttributes(TextStyle.name) },
+        getAttributes: () => { return this.editor.getAttributes(TextStyleName) },
         editor: this.editor,
       })
     }

--- a/packages/extension-ordered-list/package.json
+++ b/packages/extension-ordered-list/package.json
@@ -29,14 +29,10 @@
     "dist"
   ],
   "devDependencies": {
-    "@tiptap/core": "^2.9.0",
-    "@tiptap/extension-list-item": "^2.9.0",
-    "@tiptap/extension-text-style": "^2.9.0"
+    "@tiptap/core": "^2.9.0"
   },
   "peerDependencies": {
-    "@tiptap/core": "^2.7.0",
-    "@tiptap/extension-list-item": "^2.7.0",
-    "@tiptap/extension-text-style": "^2.7.0"
+    "@tiptap/core": "^2.7.0"
   },
   "repository": {
     "type": "git",

--- a/packages/extension-ordered-list/src/ordered-list.ts
+++ b/packages/extension-ordered-list/src/ordered-list.ts
@@ -1,6 +1,7 @@
 import { mergeAttributes, Node, wrappingInputRule } from '@tiptap/core'
-import { ListItem } from '@tiptap/extension-list-item'
-import { TextStyle } from '@tiptap/extension-text-style'
+
+const ListItemName = 'listItem'
+const TextStyleName = 'textStyle'
 
 export interface OrderedListOptions {
   /**
@@ -110,7 +111,7 @@ export const OrderedList = Node.create<OrderedListOptions>({
     return {
       toggleOrderedList: () => ({ commands, chain }) => {
         if (this.options.keepAttributes) {
-          return chain().toggleList(this.name, this.options.itemTypeName, this.options.keepMarks).updateAttributes(ListItem.name, this.editor.getAttributes(TextStyle.name)).run()
+          return chain().toggleList(this.name, this.options.itemTypeName, this.options.keepMarks).updateAttributes(ListItemName, this.editor.getAttributes(TextStyleName)).run()
         }
         return commands.toggleList(this.name, this.options.itemTypeName, this.options.keepMarks)
       },
@@ -137,7 +138,7 @@ export const OrderedList = Node.create<OrderedListOptions>({
         type: this.type,
         keepMarks: this.options.keepMarks,
         keepAttributes: this.options.keepAttributes,
-        getAttributes: match => ({ start: +match[1], ...this.editor.getAttributes(TextStyle.name) }),
+        getAttributes: match => ({ start: +match[1], ...this.editor.getAttributes(TextStyleName) }),
         joinPredicate: (match, node) => node.childCount + node.attrs.start === +match[1],
         editor: this.editor,
       })

